patches/minecraft/net/minecraft/world/level/Level.java.patch

```diff
@@ -250,6 +266,15 @@
 +      if (value > maxEntityRadius)
 +         maxEntityRadius = value;
 +      return maxEntityRadius;
-    }
- 
++   }
++
     public final boolean m_46659_() {
+       return this.f_46424_;
+    }
+@@ -900,4 +_,6 @@
+    public long m_183596_() {
+       return (long)(this.f_186455_++);
+    }
++
++   public abstract java.util.Collection<net.minecraftforge.entity.PartEntity<?>> getPartEntities();
```

Comment:
Does this count as a backwards-incompatible change? Do we care about custom Level subclasses in mods?

Reply:
They are actually quite common.....
Especially when rendering structures etc. in the world.

Reply:
Okay, then it should be a default method in `IForgeLevel`

Reply:
This entire 'fix' is not needed/not justified. There is a reason why we tell people not to add random unrelated things in PRs. No idea where this came from, our conversation on discord was related to the ID issue.

Reply:
Reverted the last commit. A new PR will be created for the `getEntities` method fix.